.github/workflows/fuzzing.yml

```diff
@@ -0,0 +1,75 @@
+name: Fuzzing
+
+on:
+  schedule:
+    # Daily midnight fuzzing
+    - cron: '0 0 * * *'
+
+jobs:
+  fuzzing:
+    name: Fuzzing
+
+    strategy:
+      matrix:
+        os: [ ubuntu-latest ]
+        go-version: [ 1.14.x ]
+    runs-on: ${{ matrix.os }}
+
+    steps:
+    - name: Install Go
+      uses: actions/setup-go@v1
+      with:
+        go-version: ${{ matrix.go-version }}
+
+    # - name: Checkout code
+    #   uses: actions/checkout@v2
+
+    # TODO env GOBIN may not be set
+    # TODO should have if condition to disable for forks?
```

Comment:
I think it's fine to let forks trigger the fuzzers. They'll only run the regression fuzzers, so there's no possibility for abuse of CPU. We had to disable forks on Azure because Azure doesn't provide the secrets (e.g. env var API keys) to runners of forks.